src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -680,7 +680,8 @@ public void testCacheAndCheckWhichAddsNewResourceLocationButKeepsSameCheckerInst
 
         final DynamicalResourceHolderCheck check = new DynamicalResourceHolderCheck();
         final String firstExternalResourceLocation = getPath("checks" + File.separator
-            + "imports" + File.separator + "import-control_one.xml");
+            + "imports" + File.separator + "importcontrol"
+            + File.separator + "InputImportControlOne.xml");
```

Comment:
There is no folder yet specifically for CheckerTest. Do just copy the input xml file outside in the 'checks' directory.